src/psfmachine/tpf.py

```diff
@@ -143,6 +144,22 @@ def remove_background_model(self, plot=False, data_augment=None):
         # keep track of which pixels comes from TPFs
         pixels_in_tpf = np.ones_like(self.row, dtype=bool)
 
+        # enheance pixel mask
+        time_corr = np.nanpercentile(bkg_flux, 20, axis=1)[:, None]
```

Comment:
Why is this 20%? Is this why stuff goes negative so often? Can you add a short docstring explainer about what's going on here?

Reply:
That part is only used to create a naive model of the background as a function of time to then find and mask out variable pixels. So it does not play into building the bkg model. 
Changing the percentile will only change which pixels to look at in the distribution each time.